globals/Party.gd

```diff
@@ -144,13 +144,13 @@ func MotivateParty(heal: int, bonus: Variant = null):
 			hero.mv += with_bonus
 			hero.mv = min(hero.mv, hero.max_mv)
 
-func GrantExp(exp: int, bonus: Variant = null):
+func GrantExp(exp_value: int, bonus: Variant = null):
```

Comment:
Change not relevant to MR.

Reply:
It's to stop it shadowing a builtin, so helps stop future bugs. Admitttedly maybe not relevant at this point.